webhooks/virtualmachine/v1alpha2/validation/virtualmachine_validator.go

```diff
@@ -349,8 +349,8 @@ func (v validator) validateNetworkInterfaceSpec(
 		networkIfCRName = fmt.Sprintf("%s-%s", vmName, interfaceSpec.Name)
 	}
 
-	for _, msg := range validation.NameIsDNSLabel(networkIfCRName, false) {
-		allErrs = append(allErrs, field.Invalid(interfacePath.Child("name"), interfaceSpec.Name, msg))
+	for _, msg := range validation.NameIsDNSSubdomain(networkIfCRName, false) {
```

Comment:
This looks fine but I misspoke yesterday - let's do this as a separate MR since it isn't related to the primary change in the MR.